radio/src/gui/480x272/lcd.cpp

```diff
@@ -557,3 +557,272 @@ void DMABitmapConvert(uint16_t * dest, const uint8_t * src, uint16_t w, uint16_t
   }
 }
 #endif
+
+//OW
+//THANKS to Adafruit and its GFX library !
+//https://learn.adafruit.com/adafruit-gfx-graphics-library
+
+void lcdDrawCircleQuarter(coord_t x0, coord_t y0, int16_t r, uint8_t corners, LcdFlags att)
+{
+  int16_t f = 1 - r;
+  int16_t ddF_x = 1;
+  int16_t ddF_y = -2 * r;
+  int16_t x = 0;
+  int16_t y = r;
+
+  if (corners >= 15) {
+	lcdDrawPoint(x0, y0 + r, att);
```

Comment:
wrong indentation

Reply:
thx for noting. Done.

this came from a space vs tab inconsistency. In order to help me avoid such mistakes in future: Can I assume that in OpenTx tabs shall be generally replaced by spaces?

Reply:
2 spaces

Reply:
thx!